vue/dynamicforms/src/components/api_consumer/consumer-logic-base.ts

```diff
@@ -119,7 +119,10 @@ abstract class ConsumerLogicBase implements APIConsumer.ConsumerLogicBaseInterfa
       this.fields[column.name] = column;
     });
     this.tableColumns = new TableColumns(UXDefinition.columns.map((col: any) => col.name), this.fields);
-    this.rows = new TableRows(this, UXDefinition.rows);
+
+    if (this.rows) this.rows.replaceRows(UXDefinition.rows);
+    else this.rows = new TableRows(this, []);
```

Comment:
if should be redundant. In constructor you already assign this.rows hence this should always be true. If it isn't, we lose reactivity anyways.